### ISSUE TITLE
Add method to submit raw JSON to clients

### ISSUE THREAD

**Issue by mvadari** [XRPLF/xrpl-py#288]
When working with advanced features that aren't on the ledger yet (e.g. NFTs, hooks, sidechains), you'd need to create your own models. But you can't use new models in the existing system.

**Comment by mDuo13** [comment]
Turns out you can do this by using `req = xrpl.models.requests.request.Request.from_xrpl(req_dict)`. However, this is a bit quirky as the `req_dict` argument needs to be in approximately WebSocket format, except with a `method` field instead of `command`. For example:;

```py
import xrpl
with (xrpl.clients.WebsocketClient("wss://s.altnet.rippletest.net:51233") as client:
    req = {
            "method": "account_info",
            "account": "rPT1Sjq2YGrBMTttX4GZHjKu9dyfzbpAYe",
            "ledger_index": "validated"
        }
    client.send(req)
```

**Comment by mvadari** [comment]
I ended up writing my own solution: https://github.com/XRPLF/xrpl-py/blob/sidechain/xrpl/asyncio/clients/client.py#L46

**Comment by mvadari** [comment]
Resolved by #313
